src/nwb2bids/_command_line_interface/_main.py

```diff
@@ -120,6 +133,13 @@ def _run_convert_nwb_dataset(
         )
         console_notification += rich_click.style(text=notification_text, fg="yellow")
 
+    if sanitization != "NONE" or sanitization != "0":
```

Comment:
```suggestion
    if handled_sanitization_level is not SanitizationLevel.NONE:
```

The original expression is `True` if `sanitization` is `"0"`, and I don't think that's what you intended. 


Using `handled_sanitization_level` will also make the intent more clear.